src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/AmendBundleNotificationHandler.java

```diff
@@ -0,0 +1,87 @@
+import static uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.MessageType.DEFAULT;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.DefaultInstructMessage;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.handlers.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service.HomeOfficeInstructService;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.service.NotificationsHelper;
+
+@Slf4j
+@Component
+public class AmendBundleNotificationHandler implements PreSubmitCallbackHandler<AsylumCase> {
+
+    private HomeOfficeInstructService homeOfficeInstructService;
+    private NotificationsHelper notificationsHelper;
+
+    public AmendBundleNotificationHandler(
+        HomeOfficeInstructService homeOfficeInstructService,
+        NotificationsHelper notificationsHelper
+    ) {
+        this.homeOfficeInstructService = homeOfficeInstructService;
+        this.notificationsHelper = notificationsHelper;
+    }
+
+    @Override
+    public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCase> callback) {
+        requireNonNull(callbackStage, "callbackStage must not be null");
+        requireNonNull(callback, "callback must not be null");
+
+        return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
+               && (callback.getEvent() == Event.SEND_DIRECTION);
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<AsylumCase> handle(
+        PreSubmitCallbackStage callbackStage, Callback<AsylumCase> callback) {
+
+        if (!canHandle(callbackStage, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+        log.info("Preparing to send {} notification to HomeOffice for event {}",
+            DEFAULT.toString(), callback.getEvent());
+
+        AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+
+        final String homeOfficeReferenceNumber = notificationsHelper.getHomeOfficeReference(asylumCase);
+
+        final String caseId = notificationsHelper.getCaseId(asylumCase);
+
+        String sendDirectionExplanation =
+            asylumCase
+                .read(SEND_DIRECTION_EXPLANATION, String.class)
+                .orElseThrow(() -> new IllegalStateException("sendDirectionExplanation is not present"));
+
+        final DefaultInstructMessage bundleInstructMessage =
```

Comment:
DefaultInstructMessage can be removed.

Reply:
Removed DefaultInstructMessage and used HomeOfficeInstruct